gulpfile.js/import/importRoadmap.js

```diff
@@ -78,10 +85,36 @@ async function importRoadmap() {
       continue;
     }
 
+    // Get full working group name from METADATA.yaml
+    let meta = null;
+    try {
+      meta = await client.fetchFile(
```

Comment:
Could have noted this in the actual importer PR: this is now blocking which makes it take 13s to run. Using `Promise.all` and splitting up the loop could bring this down by ~50% - at least that's my guess. Does not need to be done in this PR.

Reply:
Okay cool !